### DISCUSSION TITLE
Window Switcher?

### DISCUSSION THREAD

**Discussion by ghost** [quicksilver/Quicksilver#2911] [Q&A]
Is there a way to create a window switcher with the DeMinimizer plugin? I know "minimized windows" & "restore minimized windows" will open all minimized windows, but I was wondering if there was a way to get the plugin to let you choose a window to open with the qs interface. Is this possible?

**Comment by skurfer** [comment]
The User Interface Access plug-in lets you access windows. You can list them for a specific app or you can use the “Show All Windows” command (which also has a preset trigger you can configure).
